tests/gui/modules/progress-status.spec.ts

```diff
@@ -17,7 +17,20 @@
 import { expect } from 'chai';
 
 import * as settings from '../../../lib/gui/app/models/settings';
-import * as progressStatus from '../../../lib/gui/app/modules/progress-status';
+import {
+	fromFlashState,
+	ProgressStatusType,
+} from '../../../lib/gui/app/modules/progress-status';
+
+function getProgressLabel({
+	status,
+	...rest
+}: ProgressStatusType): ProgressStatusType {
+	return {
+		status,
+		...rest,
+	} as ProgressStatusType;
+}
```

Comment:
This function does nothing.

Reply:
leftover of some experiments, chai was complaining about { status: '...' } not being equal to { status: '...' } and I started going nuts